### PR TITLE
Clear memory

### PR DESCRIPTION
**Pull request by shreybirmiwal**
1) Added some documentation on adding api keys
2) Fixed how wouldn't run locally due to 'os/folderpath'
3) Added ability to clear knowledge graph within app

I added this because while playing around with Memary I wanted to be able to compare what an output with 0 knowledge graph would generate

future I want to add
- ability to edit nodes/edges manually through override  (ei if wrong info generated)
- ability to add/delete nodes
- ability to see side-by-side output by raw gpt vrs memary

https://github.com/kingjulio8238/memary/assets/67839663/7c98443e-8a38-44b2-9687-1aadd7306736

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,11 +48,45 @@ PERPLEXITY_API_KEY="YOUR_API_KEY"
...
 
+2. Remove the quotations "" because python may read escape characters '\' and skip characters
+```
+DO:
+OPENAI_API_KEY=SKxxxxxxxx
+NOT:
+OPENAI_API_KEY="SKxxxxxxxx"
+```
+
+3. How to get API keys:
```
Comment (seyeong-han) [inline README.md:59]: These are so helpful! Thanks for adding this!

File: src/agent/chat_agent.py [hunk 2]
```diff
@@ -39,6 +40,22 @@ def add_chat(self,
     def get_chat(self):
         return self.contexts
 
+    def clearMemory(self):
```
Comment (seyeong-han) [inline src/agent/chat_agent.py:43]: I think this is so useful if you wanna start from an empty graph! Sweet.

File: streamlit_app/app.py [hunk 3]
```diff
@@ -12,7 +12,10 @@
 
 # src should sit in the same level as /streamlit_app
 curr_dir = os.getcwd()
+
 parent_dir = os.path.dirname(curr_dir)
+#parent_dir = os.path.dirname(curr_dir) + '/memary' #Use this if error: src not found. Also move the '/streamlit_app/data' folder to the 'memary' folder, outside the 'src' folder.
```
Comment (seyeong-han) [inline streamlit_app/app.py:17]: Thanks for this detailed explanation!

File: streamlit_app/data/past_chat.json
Comment (seyeong-han) [inline streamlit_app/data/past_chat.json:1]: I think your case is good to add to our repository.
Is it okay to add your personal chat history?

**Review by seyeong-han — DISMISSED** [review]
Thanks for your contribution!
All the detailed README and initializing the knowledge graph for neo4j features are helpful to us.

File: streamlit_app/data/past_chat.json
Comment (shreybirmiwal) [reply streamlit_app/data/past_chat.json:1]: Sure

**Comment by shreybirmiwal** [comment]
Thanks!

File: streamlit_app/app.py (hunk 3 above)
Comment (kingjulio8238) [reply streamlit_app/app.py:17]: We will include this in the instructions. Thanks!

Comment (kingjulio8238) [reply streamlit_app/app.py:17]: This is great. Thanks!

**Comment by shreybirmiwal** [comment]
@kingjulio8238 fixed - I cleared my personal chat history

**Comment by kingjulio8238** [comment]
PR closed - see PR #26

**Pull request closed without merging**